bridgetown-core/lib/bridgetown-core/readers/defaults_reader.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Bridgetown
+  class DefaultsReader
+    attr_reader :site, :path_defaults
+
+    def initialize(site)
+      @site = site
+      @path_defaults = ActiveSupport::HashWithIndifferentAccess.new
+    end
+
+    def read
```

Comment:
This is whats causing our tests to fail.

Its looking for a `src/` directory using `bridgetown/bridgetown-core` as the root. Should we perhaps point `site.source` towards the [./bridgetown-core/test/source](https://github.com/bridgetownrb/bridgetown/tree/main/bridgetown-core/test/source) directory to fix the failing tests? I dont think this code is itself the problem, but rather we need to adjust the tests.

Reply:
Thanks for the pointer. I'm curious why this would cause the tests to break suddenly, but it should be a simple fix either way.